x-pack/plugins/infra/common/formatters/alert_link.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { modifyUrl } from '@kbn/std';
+export const getAlertLink = (timestamp: number | null) =>
+  modifyUrl('/app/logs/link-to/default/logs', ({ query, ...otherUrlParts }) => ({
```

Comment:
This generates a link to the logs app, while the function and file name suggest a generic usage. How about using a more specific name that makes this apparent?

Reply:
@weltenwort, I definitely see your point. I wanted to stick with `getAlertLink` as this function is also used in the front-end, with a _link_ notion. And on the backend side, it's used for `viewInAppUrl` 

So what do you think about something like `getLogsAppAlertUrl`? Any suggestion is welcomed!

Reply:
Yes, my question aimed at including the "logs" specificity in the name. `getLogsAppAlertUrl` sounds fine to me :+1: